Test/Unit/Service/ReplicaManagerTestable.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Test\Unit\Service;
```

Comment:
WDYT about adopting the habit to create a `Testable` subdirectory in each directory so we don't mix "real tests" with those utility classes ? Feels like we will be quickly overwhelmed otherwise.

Reply:
We could - I've been trying to stick to a naming convention of `Test` vs `Testable`. I wasn't troubled by pairs of these classes in the directory but it might be personal preference. Having it in the same directory is one less namespace import I guess but that's not a big deal. 

If we end up sharing Testables across tests I would definitely agree - but I can't imagine testing private/protected methods outside of a unit test. Integration tests should only ever test public methods IMO unless you need to mock an object to create some edge case but I haven't had need to import a Testable for that yet.